### ISSUE TITLE
Enable dropout in testing

### ISSUE THREAD

**Issue by yangjiera** [apache/incubator-mxnet#7200]
Currently, dropout is only enabled in training process (with parameters rescaled). I like to have dropout in testing, i.e., while making predictions, how could this be implemented? In this case, the rescaling in testing should be aligned with that in the training phase. 
Can I simply set "is_train = True" for forward pass in testing?

A similar problem has been solved in keras: [https://github.com/fchollet/keras/issues/1606](https://github.com/fchollet/keras/issues/1606)

Update: I find a same question here [#3821](https://github.com/dmlc/mxnet/issues/3821), which is however for R. Does anybody know a solution for Python?

**Comment by piiswrong** [comment]
https://github.com/apache/incubator-mxnet/pull/7303

Dropout will have 'always' mode after this PR

**Comment by yangjiera** [comment]
Thank you! 
I see the documentation also updated: http://mxnet.io/api/python/symbol.html#mxnet.symbol.Dropout
Nice work!
